### PR TITLE
cdc: add more metrics about output events queue time

### LINKED ISSUE #16282: Add more metrics about count of ticdc pending output events
## Question

Sometimes we can observe that incremental scans are slow. It may causes by 2 reasons:
* scan read latency is slow;
* scan is blocked by consumers.

Currently we can't distinct 2 reasons. So it's better to add more metrics.

### PR DESCRIPTION
**Pull request by hicqu**
### What is changed and how it works?

Issue Number: Close #16282 

What's Changed:

```commit-message
```

### Related changes

- [ ] PR to update `pingcap/docs`/`pingcap/docs-cn`:
- [ ] Need to cherry-pick to the release branch

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [x] Manual test (add detailed scripts or steps below)
- [ ] No code

Metrics for manual test:
图片

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

### Release note

```release-note
None
```

### CODE REVIEW

File: components/cdc/src/channel.rs [hunk 1]
```diff
@@ -294,15 +297,16 @@ impl Sink {
 }
 
 pub struct Drain {
-    unbounded_receiver: UnboundedReceiver<(CdcEvent, usize)>,
-    bounded_receiver: Receiver<(CdcEvent, usize)>,
+    unbounded_receiver: UnboundedReceiver<(Instant, CdcEvent, usize)>,
+    bounded_receiver: Receiver<(Instant, CdcEvent, usize)>,
```
Comment (CharlesCheung96) [inline components/cdc/src/channel.rs:301]: Whether the length of receiver should alse be recorded?

Comment (hicqu) [reply components/cdc/src/channel.rs:301]: Not necessary. I add another metric about sink time in one incremental scan. PTAL.

**Review by CharlesCheung96 — APPROVED** [review]

**Review by Connor1996 — APPROVED** [review]
LGTM

**Review by pingyu — APPROVED** [review]
LGTM~

**Pull request merged**